spec/index.bs

```diff
@@ -185,7 +185,7 @@ thead.stickyheader th, th.stickyheader {
 }
 
 /*
- * Darkmode colors
+ * Darkmode outputs
```

Comment:
Overzealous find/replace here and a few other places

Reply:
aaahha